swan/Dockerfile

```diff
@@ -0,0 +1,162 @@
+# Installs the extensions and packages required by SWAN
+# to adapt the base alma9 notebook image to SWAN.
+
+# FROM gitlab-registry.cern.ch/swan/docker-images/notebook
+FROM gitlab-registry.cern.ch/pesteves/imageregistry:notebook
+
+LABEL maintainer="swan-admins@cern.ch"
+ARG NB_UID="1000"
+ARG BUILD_TAG=daily
+ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
+
+RUN echo "Building systemuser image with tag ${VERSION_DOCKER_IMAGE}"
+
+# Switch to superuser to install packages
+USER root
+
+# Install required dnf packages
+RUN dnf install -y  git \
+   # gcc \
+   # gcc-c++ \
+   # kernel-devel \
+   # libcurl-openssl-devel \  -- Does not exist in dnf repos
+   # libffi-devel \
+   # ncurses-devel \
+    nano \
+    nodejs \
```

Comment:
why do we need node?

Reply:
It is no longer needed, I commented it out now.